test/built-ins/Object/prototype/isPrototypeOf/builtin.js

```diff
@@ -0,0 +1,18 @@
+// Copyright (C) 2009 the Sputnik authors. All rights reserved.
```

Comment:
```suggestion
// Copyright (C) 2019 Alexey Shvayka. All rights reserved.
```

Reply:
Please see aec7c87: this isn't a new test, but rather merged and refactored [`S15.2.4.6_A6.js`](https://github.com/tc39/test262/blob/e316773ad4715bb6a10c67698fd61776510f3e56/test/built-ins/Object/prototype/isPrototypeOf/S15.2.4.6_A6.js) and [`S15.2.4.6_A7.js`](https://github.com/tc39/test262/blob/e316773ad4715bb6a10c67698fd61776510f3e56/test/built-ins/Object/prototype/isPrototypeOf/S15.2.4.6_A7.js), authored by Sputnik team.